lib/tasks/build.js

```diff
@@ -30,9 +30,11 @@ module.exports.js = function(gulp, config) {
 			return newConfig;
 		}, {});
 
+		var useSourceMaps = config.sourcemaps === true || config.sourcemaps && config.sourcemaps.js;
```

Comment:
If I'm not mistaken, we don't need `config.sourcemaps.js` for now. Of course feel free to add it back when `config.sourcemaps` can also be an object.